usaspending_api/api_contracts/contracts/v2/autocomplete/recipient.md

```diff
@@ -32,6 +32,10 @@ This route sends a request to the backend to retrieve recipients matching the sp
                             "type": "string"
                         },
                         "description": "An array of recipient levels to filter results. E.g., ['P', 'R']"
+                    },
+                    "duns":{
+                        "type": "string",
+                        "description": "The specific duns value to filter by. E.g. '001013197' "
```

Comment:
`duns` should not be it's own new field in the request body. The existing `search_text` field should be used to search for `duns` in addition to it's existing search fields `uei` and `recipient_name`